[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/deepkick/Python_of_meteorology/blob/main/01_Thermodynamics%20of%20the%20atmosphere%20Equation%20of%20state%20of%20an%20ideal%20gas.ipynb)

# 大気の熱力学

## 01. 理想気体の状態方程式

Thermodynamics of the atmosphere Equation of state of an ideal gas



容積: $V$  
質量: $m$  
圧力: $p$  
温度: $T$  
とすると、  

\begin{equation}
pV = mRT \tag{3.1} 
\end{equation}

という関係があり、これを**理想気体の状態方程式**という。

気体の密度 $\rho$ は $\rho = \frac{m}{V}$ で定義されるので、式$(3.1)$ は

\begin{equation}
p = \rho R T \tag{3.2} 
\end{equation}

である。また比容 $\alpha = \frac{1}{\rho}$ を使えば、

\begin{equation}
p\alpha = RT \tag{3.3} 
\end{equation}

である。

分子量: $M$  
の気体が $m\ kg$ 存在するとき、その期待のキロモル数 $n$ は、

\begin{equation}
n = \frac{m}{M} \tag{3.4} 
\end{equation}

である。

**アボガドロの法則**により、同じ数の分子を含む気体は同じ圧力・同じ温度のもとでは、同じ容積を占める。

したがって、1キロモルの気体について、$\frac{pV}{T}$ という量は、気体の種類に関係なく一定である。

\begin{equation}
\frac{pV}{T} = mR = MR = R^* \tag{3.5} 
\end{equation}

温度: $T = 273.15\ K$  
気圧: $p = 1013.25\ hPa = 1気圧$  
のとき、1キロモルの気体の容積は、  
容積: $V = 22.4135\ m^3$  
であるから、式$(3.5)$ により、  

\begin{equation}
R^* = 8.3143 × 10^3\ JK^{-1}kmol^{-1}
\end{equation}

であり、これを一般気体定数という。

混合気体の平均分子量 $\overline{M}$ 、$1 kg$ の混合気体の気体定数 $\overline{R}$ を定義すると、

\begin{equation}
\overline{R} = \frac{R^*}{\overline{M}} \tag{3.9} 
\end{equation}

\begin{equation}
p\alpha = \overline{R}T \tag{3.10} 
\end{equation}

である。

## 1. 地球大気について

$Table 1$ 

地表付近の大気組成（「一般気象学　第２版補訂版」 p13, 表1.4 より）

|  成分  |  分子式  |  分子量  |  容積比（存在比率%）  |  重量比（存在比率%）  |
| ---- | ---- | ---- | ---- | ---- |
|  窒素分子  |  $N_2$  |  $28.01$  |  $78.088$  |  $75.527$  |
|  酸素分子  |  $O_2$  |  $32.00$  |  $20.949$  |  $23.413$  |
|  アルゴン  |  $Ar$  |  $39.94$  |  $0.93$  |  $1.282$  |
|  二酸化炭素  |  $CO_2$  |  $44.01$  |  $0.03$  |  $0.0456$  |
|  一酸化炭素  |  $CO$  |  $28.01$  |  $1×10^{-5}$  |  $1×10^{-5}$  |
|  ネオン  |  $Ne$  |  $20.18$  |  $1.8×10^{-3}$  |  $1.25×10^{-3}$  |
|  ヘリウム  |  $He$  |  $4.00$  |  $5.24×10^{-4}$  |  $7.24×10^{-5}$  |
|  メタン  |  $CH_4$  |  $16.05$  |  $1.4×10^{-4}$  |  $7.25×10^{-5}$  |
|  クリプトン  |  $Kr$  |  $83.7$  |  $1.14×10^{-4}$  |  $3.30×10^{-4}$  |
|  一酸化二窒素  |  $N_2O$  |  $44.02$  |  $5×10^{-5}$  |  $7.6×10^{-5}$  |
|  水素分子  |  $H_2$  |  $2.02$  |  $5×10^{-5}$  |  $3.48×10^{-6}$  |
|  オゾン  |  $O_3$  |  $48.0$  |  $2×10^{-6}$  |  $3×10^{-6}$D  |
|  水蒸気  |  $H_2O$  |  $18.02$  |  不定  |  不定  |

### (1) 容積比を使って、乾燥空気の平均分子量$\ M_d \ $を算出する。

$Table1$ より、大気組成のうち「窒素分子・酸素分子・アルゴン」のみを考慮するものとした。

乾燥空気の分子量: $M_d\ =\ $(窒素分子量$\ *\ $窒素分子の容積比)$\ +\ $(酸素分子量$\ *\ $酸素分子の容積比)$\ +\ $(アルゴン分子量$\ *\ $アルゴン分子の容積比)


In [3]:
# 窒素分子・酸素分子・アルゴンそれぞれの容積比
N_2_v = 78.088 * (10 ** -2)
O_2_v = 20.949 * (10 ** -2)
Ar_v = 0.93 * (10 ** -2)

# 窒素分子・酸素分子・アルゴンそれぞれの分子量
N_2_mol_w = 28.01
O_2_mol_w = 32.00
Ar_mol_w = 39.94

M_d = (N_2_mol_w * N_2_v) + (O_2_mol_w * O_2_v) + (Ar_mol_w * Ar_v)
# M_d
print('乾燥空気の平均分子量: ', M_d)

乾燥空気の平均分子量:  28.947570799999998


### (2) 重量比を使って、乾燥空気の平均分子量$\ M_d \ $を算出する。

$Table1$ より、大気組成のうち「窒素分子・酸素分子・アルゴン」のみを考慮するものとした。

乾燥空気の分子量: $M_d\ =\ $(窒素分子量$\ *\ $窒素分子の容積比)$\ +\ $(酸素分子量$\ *\ $酸素分子の容積比)$\ +\ $(アルゴン分子量$\ *\ $アルゴン分子の容積比)

\begin{eqnarray}
M_d &=& \frac{1}{\frac{窒素分子の重量比}{窒素分子量} + \frac{酸素分子の重量比}{酸素分子量} + \frac{アルゴン分子の重量比}{アルゴン分子量}} \nonumber \\
\end{eqnarray}



In [4]:
# 窒素分子・酸素分子・アルゴンそれぞれの重量比
N_2_w = 75.527 * (10 ** -2)
O_2_w = 23.413 * (10 ** -2)
Ar_w = 1.282 * (10 ** -2)

M_d = 1 / ((N_2_w / N_2_mol_w) + (O_2_w / O_2_mol_w) + (Ar_w / Ar_mol_w))
# M_d
print('乾燥空気の平均分子量: ', M_d)

乾燥空気の平均分子量:  28.90019517947606


### (3) $\ 1kg\ $の乾燥空気に対する気体定数 $\ R_d \ $ を算出する。

一般気体定数 $R^* = 8.3143 × 10^3\ JK^{-1}kmol^{-1}$

\begin{eqnarray}
R_d &=& \frac{R^*}{M_d} \nonumber \\
&=& \frac{8.3143 * 10^3}{28.9} \nonumber \\
&≒& 287\ JK^{-1}kg^{-1} \nonumber \\
\end{eqnarray}

## 2. 金星大気について

金星の大気では、容積比で、$CO_2$ が95%、$N_2$ が5%を占めているとする。$C, O, N$ それぞれの原子量は $12, 16, 14$ とする。

$Table 2$ 

金星の大気組成

|  成分  |  分子式  |  分子量  |  容積比（存在比率%）  |
| ---- | ---- | ---- | ---- |
|  二酸化炭素分子  |  $CO_2$  |  12 + 16 × 2 = 44  |  $95.0$  |
|  窒素分子  |  $N_2$  |  14 × 2 = 28  |  $5.0$  |

### (1) 金星大気の平均分子量 $M_dv$ を算出する。

\begin{eqnarray}
M_dv &=& 44 * 0.95 + 28 * 0.05 \nonumber \\
&=& 41.8 + 1.4 \nonumber \\
&=& 43.2 \nonumber \\
\end{eqnarray}

### (2) 金星大気 $\ 1kg\ $の気体定数 $\ R_dv \ $ を算出する。

\begin{eqnarray}
R_dv &=& \frac{R^*}{M_dv} \nonumber \\
&=& \frac{8.3143 * 10^3}{43.2} \nonumber \\
&=& 192.46\ JK^{-1}kg^{-1} \nonumber \\
\end{eqnarray}

#### 参考文献　
* 「一般気象学　第2版補訂版」 小倉義光著